table.csv

```diff
@@ -19,9 +19,13 @@ keccak-256,                     multihash,      0x1b,
 keccak-384,                     multihash,      0x1c,
 keccak-512,                     multihash,      0x1d,
 blake3,                         multihash,      0x1e,           BLAKE3 has a default 32 byte output length. The maximum length is (2^64)-1 bytes.
+sha2-224,                       multihash,      0x1f,           aka SHA-224; specified by FIPS 180-4.
+sha2-384,                       multihash,      0x20,           aka SHA-384; specified by FIPS 180-4.
 dccp,                           multiaddr,      0x21,
 murmur3-128,                    multihash,      0x22,
 murmur3-32,                     multihash,      0x23,
+sha2-512-224,                   multihash,      0x24,           aka SHA-512/224; specified by FIPS 180-4.
```

Comment:
IMO these should be up in the `0x1012` range. The single byte range is precious. `sha2-224` is unlikely to get wide usage too so I'd prefer that to be higher as well. I can see the case for 384 though.

fwiw I don't think FIPS 180-4 is a strong claim to their level of importance at this stage, in crypto terms it's ancient, if you're making new software then it's not highly likely that you're reaching for it as your standard. And FIPS-202 is all about SHA3 I believe.

Reply:
Oh I'm definitely not claiming a FIPS number expresses anything about their importance :) That's just there to remove any possible ambiguity about the naming vs what the thing is.